wordssite/words/models.py

```diff
@@ -0,0 +1,39 @@
+import datetime
+import uuid
+from django.db import models
+from django.utils import timezone
+from django.conf.global_settings import LANGUAGES
+
+
+class BaseModel(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        abstract = True
+
+    
+class UUIDBaseModel(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False, unique=True)
+
+
+    class Meta:
+        abstract = True
+
+class Word(UUIDBaseModel):
+    word_text = models.CharField(max_length=200)
+
+    def __str__(self):
+        return self.word_text
+
+
+class Translation(UUIDBaseModel):
+    word = models.ForeignKey(Word, on_delete=models.CASCADE)
+    translation_text = models.CharField(max_length=200)
+    language_text = models.CharField(max_length=7, choices=LANGUAGES)
```

Comment:
I was actually thinking that Language should be yet another model that inherits from BaseModel, and it would have a full title and maybe a short code (kida like LT, EN etc.) for starters. And a Translation would have a FK pointing to it.
Also, let's add all the models to admin.

Reply:
I don't understand the practice of it- why it is better to have Language be a separate model and not just an attribute?